src/sql/workbench/contrib/notebook/browser/cellViews/code.component.ts

```diff
@@ -277,6 +279,48 @@ export class CodeComponent extends CellView implements OnInit, OnChanges {
 			DOM.getContentWidth(this.codeElement.nativeElement),
 			DOM.getContentHeight(this.codeElement.nativeElement)));
 		this._editor.setHeightToScrollHeight(false, this._cellModel.isCollapsed);
+		this.horizontalScrollbar();
+	}
+
+	/**
+	 * Horizontal Scrollbar function will ensure we only calculate and trigger this if word wrap is off and it is a markdown cell
+	 * This will adjust the horizontal scrollbar to either a fixed position at the bottom of the viewport (visible area)
+	 * or it will set it to the bottom of the markdown editor if it is in the viewport (visible area)
+	 */
+	public horizontalScrollbar(): void {
+		if (this._configurationService.getValue('editor.wordWrap') === 'off' && this.cellModel.cellType !== CellTypes.Code && this.cellModel.source.length > 0) {
```

Comment:
Should this logic be triggered for every markdown editing type (and regardless of editing/preview mode)?

Reply:
I can discuss this with Julie, as I believe that users will also see these problems when editing markdown as well. They would have to scroll down every time to see the horizontal scrollbar if the bottom of the editor is not visible.

Reply:
Ok. What about wysiwyg?

Reply:
Discussed offline. WYSIWYG word wrap support is low pri as of now.